draft-ietf-tls-esni.md

```diff
@@ -963,6 +967,17 @@ verifying the server's identity in its certificate.
 odd, and probably some precise rules about handling ECHO and no
 SNI uniformly?]]
 
+## Padding
+
+Variations in the length of the ciphertext version of the ClientHelloInner
+could defeat the purpose of ECHO, if those expose the chosen server_name field
```

Comment:
Do we need to update GREASE if we are rounding the packet sizes to multiples of 32 octets? If we want GREASE to look like ECHO then it should have the same external properties.

Reply:
Yep -- there's a TODO in the GREASE section.